### ISSUE TITLE
LDAP Support

### ISSUE THREAD

**Issue by yalios** [taigaio/taiga-docker#48]
**Is your feature request related to a problem? Please describe.**
Not a problem related

**Describe the solution you'd like**
It will be great to see an option for LDAP support included in the official taigaio docker image.

**Describe alternatives you've considered**
The are some good contributions from the communicate that offer that option, but they are not the latest taiga version and it will be great to see that option on the official supported taigaio

**Additional context**
Thanks for your efforts and making this great app available to all!

**Comment by gabrieletassoni** [comment]
Yes, I was looking for the info about compatibility with external LDAP plugins or the integration of LDAP into taiga and from what I understand, it's still not the time to upgrate to taiga 6 for me, until Taig will include LDAP support or the taiga-contrib-ldap-auth-ext will be compatible with Taiga 6.

**Comment by yamila-moreno** [comment]
Hi!

Currently, the docker images contain the plugins maintained by Taiga. We plan to improve the system to add other plugins.

Meanwhile you should easily extend the official images creating your `Dockerfile`:

```
FROM taigaio/taiga-back:latest

# your additions
```

Cheers!

**Comment by kota65535** [comment]
Any progress?
